src/main/java/com/shmkane/sellstick/PlayerListener.java

```diff
@@ -207,21 +204,21 @@ public boolean isSellStick(Player p, PlayerInteractEvent e) {
      *
      * @param p Player that clicked the chest
      * @param e On a player interact event
-     * @return True if the item in hand was a sellstick && player clicked a chest
+     * @return True if the item in hand was a sellstick &amp player clicked a chest

```

Comment:
🤔

Reply:
Replaced with &amp as the java doc plugin was throwing errors in parsing that as HTML